src/components/IDO/Ido.tsx

```diff
@@ -0,0 +1,165 @@
+      <Grid container className={classes.header}>
+        <Typography component='h1'>IDO</Typography>
+      </Grid>
+      <Grid className={classes.main}>
+        <Grid container className={classes.root} direction='column'>
+          <Typography component='h1' className={classes.label}>
+            Deposit your SOL
+          </Typography>
+          <Box className={classes.tokenComponentTextContainer}>
+            <Typography className={classes.tokenComponentText}>Est.: 56.0278$ </Typography>
+            <Typography className={classes.tokenComponentText}>
+              Balance:{' '}
+              {tokenFromIndex !== null
+                ? printBN(tokens[tokenFromIndex].balance, tokens[tokenFromIndex].decimals)
+                : '0'}
+            </Typography>
+          </Box>
+          <ExchangeAmountInput
+            value={amountFrom}
+            key={swap?.toString()}
+            decimal={tokenFromIndex !== null ? tokens[tokenFromIndex].decimals : 6}
+            className={
+              swap !== null
+                ? `${classes.amountInput} ${classes.amountInputDown}`
+                : `${classes.amountInput}`
+            }
+            style={{
+              transform: swap !== null ? (swap ? 'translateY(0px)' : 'translateY(0px)') : ''
+            }}
+            setValue={value => {
+              if (value.match(/^\d*\.?\d*$/)) {
+                setAmountFrom(value)
+                updateEstimatedAmount(value)
+              }
+            }}
+            placeholder={'0.0'}
+            onMaxClick={() => {
+              if (tokenToIndex !== null && tokenFromIndex !== null) {
+                setAmountFrom(
+                  printBN(tokens[tokenFromIndex].balance, tokens[tokenFromIndex].decimals)
+                )
+                updateEstimatedAmount(
+                  printBN(tokens[tokenFromIndex].balance, tokens[tokenFromIndex].decimals)
+                )
+              }
+            }}
+            tokens={tokens}
+            current={tokenFromIndex !== null ? tokens[tokenFromIndex] : null}
+            onSelect={(name: string) => {
+              setTokenFromIndex(
+                tokens.findIndex(token => {
+                  console.log('URI', token.logoURI)
+                  return name === token.symbol
+                })
+              )
+            }}
+          />
+          <Box className={classes.depositedAmountContainer}>
+            <Typography className={classes.tokenComponentText}>Deposited amount: </Typography>
+            <Box className={classes.depositedAmounts}>
+              <img src='../../public/logoGrey.png' />
+              <Box className={classes.depositedAmountTextContainer}>
+                <Typography className={classes.depositedAmountMainText}>46.643 xUSD </Typography>
+                <Box className={classes.depositedAmountSecondaryTextContainer}>
+                  <Typography className={classes.depositedAmountSecondaryText}>
+                    47.43 USD{' '}
```

Comment:
don't hardcode any data in any component if not necessary - you have stories for that